pinot-core/src/main/java/org/apache/pinot/core/common/datablock/BaseDataBlock.java

```diff
@@ -221,7 +221,7 @@ public BaseDataBlock(ByteBuffer byteBuffer)
    * @param colId column ID
    * @return the length to extract from variable size buffer.
    */
-  protected abstract int positionCursorInVariableBuffer(int rowId, int colId);
+  protected abstract int computePositionCursorInVariableBuffer(int rowId, int colId);
```

Comment:
Don't change this method name. It does not return the offset, but actually position the cursor

Reply:
yeah and return the length to read out of the var-byte buffer. let me rethink the naming